sdcm/logcollector.py

```diff
@@ -1233,6 +1235,29 @@ def get_gce_instances_by_testid(self):
         if self.params["use_cloud_manager"]:
             self.find_and_append_cloud_manager_instance_to_collecting_nodes()
 
+    def create_collecting_nodes(self):
+        try:
+            provisioners = provisioner_factory.discover_provisioners(backend=self.backend, test_id=self.test_id)
+            instances = sum([provisioner.list_instances() for provisioner in provisioners], [])
+            collecting_nodes = [CollectingNode(name=instance.name,
+                                               ssh_login_info={
+                                                   "hostname": instance.public_ip_address,
+                                                   "user": instance.user_name,
+                                                   "key_file": f"~/.ssh/{instance.ssh_key_name}"},
+                                               instance=instance,
+                                               global_ip=instance.public_ip_address,
+                                               tags=instance.tags) for instance in instances]
+            self.db_cluster = [c_node for c_node in collecting_nodes if c_node.tags.get(
+                "NodeType") in ("scylla-db", "oracle-db")]
+            self.monitor_set = [c_node for c_node in collecting_nodes if c_node.tags.get("NodeType") == "monitor"]
+            self.loader_set = [c_node for c_node in collecting_nodes if c_node.tags.get("NodeType") == "loader"]
+            self.kubernetes_set = [c_node for c_node in collecting_nodes
+                                   if c_node.tags.get("NodeType") == "k8s"]  # why aws and gce uses 'loader'? Bug?
```

Comment:
I don't think we have a this `NodeType==k8s` defined anywhere
@vponomaryov correct me if i'm wrong.